### PR TITLE
[miio] add new devices

### PR DESCRIPTION
**Pull request by marcelrv**
* Add  Xiaomi Mi Smart Plug (chuangmi.plug.hmi205)
* Add 2nd gen airmonitor

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.miio/README.md [hunk 1]
```diff
@@ -204,6 +206,21 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
...
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| power            | Switch  | Power                               |
+| aqi              | Number  | Air Quality Index                   |
+| battery          | Number  | Battery                             |
+| usb_state        | Switch  | USB State                           |
+| time_state       | Switch  | Time State                          |
+| night_state      | Switch  | Night State                         |
+| night_begin      | Number  | Night Begin Time                    |
+| night_end        | Number  | USB State                           |
```
Comment (wborn) [inline addons/binding/org.openhab.binding.miio/README.md]: USB State -> Night End Time

File: addons/binding/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -1388,6 +1414,22 @@ Number night_begin "Night Begin Time" (G_airmonitor) {channel="miio:basic:airmon
...
+```
+Group G_airmonitor "Mi Air Quality Monitor 2gen" <status>
+Switch power "Power" (G_airmonitor) {channel="miio:basic:airmonitor:power"}
+Number aqi "Air Quality Index" (G_airmonitor) {channel="miio:basic:airmonitor:aqi"}
+Number battery "Battery" (G_airmonitor) {channel="miio:basic:airmonitor:battery"}
+Switch usb_state "USB State" (G_airmonitor) {channel="miio:basic:airmonitor:usb_state"}
+Switch time_state "Time State" (G_airmonitor) {channel="miio:basic:airmonitor:time_state"}
+Switch night_state "Night State" (G_airmonitor) {channel="miio:basic:airmonitor:night_state"}
+Number night_begin "Night Begin Time" (G_airmonitor) {channel="miio:basic:airmonitor:night_begin"}
+Number night_end "USB State" (G_airmonitor) {channel="miio:basic:airmonitor:night_end"}
```
Comment (wborn) [inline addons/binding/org.openhab.binding.miio/README.md]: USB State -> Night End Time

File: addons/binding/org.openhab.binding.miio/README.md [hunk 3]
```diff
@@ -909,6 +926,15 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | led              | Switch  | Wifi led                            |
 
 
+### Mi Smart Plug (<a name="chuangmi-plug-hmi205">chuangmi.plug.hmi205</a>) Channels
+
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| power            | Switch  | Power                               |
+| temperature      | Number  | Temperature                         |
```
Comment (wborn) [inline addons/binding/org.openhab.binding.miio/README.md]: Any plans for using UoM with this binding? :slightly_smiling_face:

Comment (marcelrv) [reply addons/binding/org.openhab.binding.miio/README.md]: hmmm, not really high on my list of wishes to be honest.

Comment (marcelrv) [reply addons/binding/org.openhab.binding.miio/README.md]: thanks

File: addons/binding/org.openhab.binding.miio/README.md (hunk 1 above)
Comment (marcelrv) [reply addons/binding/org.openhab.binding.miio/README.md]: thanks

**Pull request merged**